src/pages/home/_hero.js

```diff
@@ -1,240 +1,174 @@
+    'Tight spreads',
+    'Sharp prices',
+    '24x7 trading',
+    '100+ tradeable assets',
+    '20+ of experience',
+]
+const TypeWriter = styled(Header)`
+    min-height: 7.2rem;
+`
+const HeroContainer = styled(CssGrid)`
+    grid-template-columns: repeat(12, 1fr);
+    width: 100%;
+    grid-column-gap: 2.4rem;
+    grid-template-areas:
+        'details details details details details video video video video video video video'
+        'button button button button button button button button button button button button';
 
-const ButtonWrapper = styled(Flex)`
-    margin-top: 9.6rem;
-    height: 40px;
-
-    ${Button} {
-        font-size: 1.6rem;
-    }
-    @media ${device.laptop} {
-        text-align: center;
-    }
     @media ${device.tabletL} {
-        text-align: left;
+        grid-template-columns: repeat(1, 1fr);
+        grid-template-areas:
+            'details'
+            'video'
+            'button';
     }
 `
 
-const StyledVideo = styled.video`
-    opacity: 0.5;
-    max-height: 82.7rem;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-    width: 100%;
-    height: 100%;
-
-    /* for edge */
-    @supports (object-fit: fill) {
-        object-fit: fill;
-        top: 0%;
-        left: unset;
-        transform: unset;
-    }
+const Details = styled(Box)`
+    grid-area: details;
 `
-const HeroButton = styled(LinkButton)`
-    height: 4.8rem;
-    display: flex;
-    align-items: center;
+const ButtonWrapper = styled(Box)`
+    grid-area: button;
 
     @media ${device.tabletL} {
-        width: 27rem;
-        margin: 0 auto;
-        display: flex;
-        font-size: 1.75rem;
-        justify-content: center;
+        margin-top: 3rem;
     }
 `
+const ImageWrapper = styled(Box)`
+    grid-area: video;
+    margin-top: 4rem;
 
-const CheckBoxText = styled(Text)`
-    @media ${device.desktopL} {
-        font-size: 2.52rem;
-    }
     @media ${device.tabletL} {
-        font-size: 2rem;
+        min-height: 25rem;
+        margin-top: 0;
     }
 `
-const CheckMarkBullet = ({ children }) => (
-    <li>
-        <Flex jc="unset" ai="center">
-            <CheckMark />
-            <CheckBoxText ml="1rem" size="2.4rem" lh="1.5" color="white">
-                {children}
-            </CheckBoxText>
-        </Flex>
-    </li>
-)
-CheckMarkBullet.propTypes = {
-    children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
-}
-
-const UList = styled.ul`
-    list-style-type: none;
-`
-
-const CheckMarkList = ({ children }) => <UList>{children}</UList>
-CheckMarkList.propTypes = {
-    children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
-}
-
-CheckMarkBullet.propTypes = {
-    children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
-}
-
 export const Hero = () => {
-    useLazyVideo()
+    const data = useStaticQuery(query)
+    const subtitle = localize('Trade forex, commodities, synthetic and stock indices')
```

Comment:
when its applying from js to jsx localize need to use <Localize /> instead

Reply:
Sure